databases/week1/hyf_lesson1_homework_queries.sql

```diff
@@ -0,0 +1,39 @@
+USE hyf_lesson1;
+--1
+SELECT COUNT(*) AS 'count task '

```

Comment:
`*` is a much general approach. Preferably count(id) is more suited  and id's cannot be null